inc/ConfigFileParser.hpp

```diff
@@ -3,59 +3,59 @@
 /* ====== LIBRARIES ====== */
 
 #include "ConfigFile.hpp"
+#include "utilities.hpp"
 #include <algorithm>
 #include <fstream>
+#include <iostream>
 #include <set>
 #include <stack>
 #include <string>
-#include <iostream>
 
 /* ====== DEFINITIONS ====== */
 
-#define SERVER 1
-#define LOCATION 2
+enum Block { ServerBlock, LocationBlock };
+#define WHITESAPCE " \t\n\v\f\r"
```

Comment:
Better to use a const variable, see https://webkit.org/code-style-guidelines/#names-const-to-define

Reply:
True. Gabriel also mentioned it and replaced with his approach, by using a static const char* const whitespace variable.